.github/workflows/deploy-prod.yml

```diff
@@ -0,0 +1,33 @@
+# .github/workflows/deploy-prod.yml
+
+name: Deploy to Live Channel
+
+on:
+  push:
+    branches:
+      - master
+
+jobs:
+  preview:
+    runs-on: ubuntu-latest
+    defaults:
+      run:
+        working-directory: ./example
+
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-java@v1

```

Comment:
Same issue as above; I don't think setting up Java is necessary.

Reply:
Fixed!